### DISCUSSION TITLE
Restore Faces Removed

### DISCUSSION THREAD

**Discussion by Krader13** [zero01101/openOutpaint#288] [General]
Why was Restore Faces option removed from this commit?  I put it back in my local copy and it works well.

https://github.com/zero01101/openOutpaint/commit/c252ed6ea1f55ae042f2646949a09f044a4b9628

@zero01101

**Comment by zero01101** [comment]
hey, incredibly sorry for the delay - to be frank, i removed it because it seemed to be a _really_ minimally used option, to the point where auto1111 himself moved it out of regular *2img interfaces in lieu of an option and possible user-set quicksetting, plus the UI was getting a bit cluttered with all the more recent additions :/
